driver/modern_bpf/programs/tail_called/events/syscall_dispatched_events/mount.bpf.c

```diff
@@ -73,15 +73,15 @@ int BPF_PROG(mount_x,
 
 	/* Parameter 2: dev (type: PT_CHARBUF) */
 	unsigned long source_pointer = extract__syscall_argument(regs, 0);
-	auxmap__store_charbuf_param(auxmap, source_pointer, USER);
+	auxmap__store_charbuf_param(auxmap, source_pointer, MAX_PARAM_SIZE, USER);
```

Comment:
`MAX_PATH` ? Or is there any specific use case here which I am not aware of?

Reply:
this could be also a dummy string quoting the manual:
> (which is often a pathname referring to a device, but can also be the pathname of a directory or file, or a dummy string)

BTW I think we could limit it as a pathname :thinking:

Reply:
Changed it :)

Reply:
Thanks!